.travis.yml

```diff
@@ -21,6 +48,8 @@ before_script:
   - composer require --dev typo3/cms="$TYPO3_VERSION"
   # Restore composer.json
   - git checkout composer.json
+  - export UNIT_BOOTSTRAP='.Build/vendor/nimut/testing-framework/res/Configuration/UnitTestsBootstrap.php'
```

Comment:
Doesn't the Configuration/UnitTests.xml not include the bootstrapping ?

Reply:
This configuration was required previously. After refactoring the tests this is no longer required. Using "Configuration/UnitTests.xml" is enough now.